corehq/apps/app_manager/detail_screen.py

```diff
@@ -102,12 +102,12 @@ def __init__(self, app, module, detail, column, sort_element=None,
 
     @property
     def locale_id(self):
-        if not is_sort_only_column(self.column):
+        if is_sort_only_column(self.column):
+            return self.detail.sort_elements[0].display
```

Comment:
You should add a new function to `id_strings.py` for the sort only column and then you can map the ID string to the actual text in `app_strings.py`. Search for references to `detail_column_header_locale` to see how that works.